server/src/main/scala/sbt/server/SbtClientHandler.scala

```diff
@@ -150,7 +150,7 @@ class SbtClientHandler(
           // Now clean out the request handler
           readLineRequests -= serial
           confirmRequests -= serial
-        case None => // TODO - error
+        case _ => // TODO - error
```

Comment:
Mark harrah has me paranoid of using fall-through pattern because you loose any case you may not catch if the types change.   Just curious what the reasoning is here on your end.